server/app/helpers/auth.js

```diff
@@ -0,0 +1,21 @@
+import jwt from 'jsonwebtoken'
+import { config } from 'dotenv'
+
+config();
+
+const secretKey = process.env.SECRET_KEY;
+
+class Auth {
+	static generateMailToken(payload) {
```

Comment:
Please check your tab-size. It should be 2 spaces rather than 2 tabs. The tab size is enormous.